sysrib/server_test.go

```diff
@@ -627,33 +681,102 @@ func TestServer(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.desc, func(t *testing.T) {
+			grpcServer := grpc.NewServer()
```

Comment:
why make a server for every test case? you create the server once and delete /interfaces/ at the start of the test case

Reply:
Good point.. created TODO, I still need to implement deleting interfaces. This would be a good test of this ability.

Reply:
i doubt deleting interface is something we something we ever need to care about. (when would a user delete an interface) if restart the sysrib between test cases it will need see the delete

Reply:
Got it, updated TODO's comment for this.